packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -16,6 +17,27 @@ interface State {
   data: DataFrame[];
 }
 
+const getStyles = () => {
+  return {
+    gfTableInputCsv: css`
+      position: relative;
+      textarea {
+        height: 100%;
+        width: 100%;
+      }
+      footer {
+        position: absolute;
+        bottom: 15px;
+        right: 15px;
+        border: 1px solid #222;
+        background: $online;
+        padding: 1px 4px;
+        font-size: 80%;
```

Comment:
In general we should not use any statically defined sizing/colors. Insead, please use theme variables :)

Reply:
In a situation where the sizing is not defined in the theme variables file, is it advisable to use the statically defined sizing regardless?